hbase_sql/src/test/java/com/splicemachine/stream/StreamListenerIT.java

```diff
@@ -0,0 +1,102 @@
+package com.splicemachine.stream;
+
+import com.splicemachine.derby.test.framework.*;
+import com.splicemachine.test.LongerThanTwoMinutes;
+import com.splicemachine.test.SerialTest;
+import com.splicemachine.test.SlowTest;
+import org.junit.*;
+import org.junit.experimental.categories.Category;
+import org.junit.rules.RuleChain;
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.*;
+
+@Category({LongerThanTwoMinutes.class, SlowTest.class, SerialTest.class})
+public class StreamListenerIT {
```

Comment:
How long does this new IT take in jenkins?

Reply:
It timed out after 280s in my laptop, is it possible to test this feature with a lighter weight test?

Reply:
I've tested it locally, it failed only on 6th or 7th attempt. Takes about 2,5 mins.
On jenkins it's worse, failed on second attempt:

`2021-06-08|21:19:43,071 [INFO] Running com.splicemachine.stream.StreamListenerIT
2021-06-08|21:28:17,632 [INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 514.474 s - in com.splicemachine.stream.StreamListenerIT

2021-06-09|19:35:09,450 [ERROR] testThrottlingNotBlocks(com.splicemachine.stream.StreamListenerIT)  Time elapsed: 246.612 s  <<< FAILURE!
java.lang.AssertionError: expected null, but was:<The statement has been cancelled or timed out.>
	at com.splicemachine.stream.StreamListenerIT.testThrottlingNotBlocks(StreamListenerIT.java:98)`

successful execution took about 5 mins

I've removed the test for now and will prepare better test in DB-12211